addons/website/static/src/js/backend/dashboard.js

```diff
@@ -320,10 +326,16 @@ var Dashboard = AbstractAction.extend(ControlPanelMixin, {
                 self.on_website_button($(ev.target).data('website-id'));
             });
         }
+
+        var $buttons = $(QWeb.render("website.GoToButtons"));
+        $buttons.find('a.js_go_to_website').addBack('a.js_go_to_website').click(function (ev) {
```

Comment:
- Just $buttons.click(
- `.on('click', this.on_go_to_website.bind(this))`

Reply:
Done, as discussed was to ensure targeting the button if the DOM change someday but should not think that far